src/main/kotlin/org/jetbrains/research/testspark/helpers/PsiClassHelper.kt

```diff
@@ -0,0 +1,40 @@
+package org.jetbrains.research.testspark.helpers
+
+import com.intellij.psi.PsiClass
+import com.intellij.psi.PsiModifier
+import org.jetbrains.research.testspark.core.data.ClassType
+
+/**
+ * Helper class for processing PsiClass objects and determining their types.
+ */
+object PsiClassHelper {
```

Comment:
This methods should be covered in `PsiHelper`, because we no longer use `PsiClasses` outside of the `JavaPsiHelper`